CefSharp.WinForms.Example/Handlers/MenuHandler.cs

```diff
@@ -10,16 +10,24 @@ void IContextMenuHandler.OnBeforeContextMenu(IWebBrowser browserControl, IBrowse
         {
             //To disable the menu then call clear
             // model.Clear();
+
+            bool removed = model.Remove(132); // Remove "View Source" option
```

Comment:
Will need to add a new `enum` based on http://magpcss.org/ceforum/apidocs3/projects/%28default%29/cef_menu_id_t.html